deploy/README.md

```diff
@@ -1,72 +1,81 @@
-
-
-2. Use the running `admin` container to modify any config values, and also reload the hub when complete:
-  - [GitHub - Gaia Admin README.md](https://github.com/stacks-network/gaia/blob/master/admin/README.md)
-
-  ```
-  $ export API_KEY="hello"
-  $ curl -s \
-        -H "Authorization: bearer $API_KEY" \
-        -H 'Content-Type: application/json' \
-        -X POST \
-        --data-raw '{"serverName": "myserver"}' \
-    http://localhost:8009/v1/admin/config
-
-  $ curl -s \
-        -H "Authorization: bearer $API_KEY" \
-        -X POST \
-    http://localhost:8009/v1/admin/reload
-
-  $ curl -s \
-        -H "Authorization: bearer $API_KEY" \
-    http://localhost:8009/v1/admin/config | jq
-  ```
+# Deploy your GAIA Hub
+
+Step by step instructions to setup you own GAIA Hub can be found in the following links. General configuration and platform agnostic setup is explained further below.
+
+* [GAIA Hub on Linux](setup_linux.md)
+* [GAIA Hub on Mac](setup_mac.md)
+
+## Configuring your GAIA HUB
+
+### Configuration files
+The following configuration files exist in the folder `deploy/configs/gaia/`:
+
+```
+admin-config.json
+hub-config.json
+reader-config.json
+```
+
+### GAIA Admin Service
+You can also use the GAIA Admin Service to remotely administer it with an API Key. It will require you to install `npm` with a `apt install npm`. Once `npm` is installed you can continue with the steps in the [GAIA Admin Service](https://github.com/stacks-network/gaia/blob/master/admin/README.md).
+
+### Driver Selection
+
+The Gaia hub currently supports the following drivers:
+
+```
+'aws' == Amazon S3
+'azure' == Azure Blob Storage
+'disk' == Local disk
+'google-cloud' === Google Cloud Storage
+```
+
+Set the driver you wish to use in your [config.json](https://github.com/stacks-network/gaia/blob/master/hub/config.sample.json) file with the `driver` parameter. Many drivers additionally accept the `bucket` parameter, which controls the bucket name that files should be written to.
+
+These driver may require you to provide additional credentials for performing writes to the backends. See `config.sample.json` for fields for those credentials. In some cases, the driver can use a system configured credential for the backend (e.g., if you are logged into your AWS CLI account, and run the hub from that environment, it won't need to read credentials from your `config.json`).
+
+*Note:* The disk driver requires a *nix like filesystem interface, and will not work correctly when trying to run in a Windows environment.
+
+### Note on SSL
+We *strongly* recommend that you deploy your Gaia hub with SSL enabled. Otherwise, the tokens used to authenticate with the Gaia hub may be stolen by attackers, which could allow them to execute writes on your behalf.  
+Configuration options are available to run the hub with an `https` Node.js server.  
+Otherwise, a reverse proxy web server such as nginx or Apache can be used.  
+
+### Require Correct Hub URL
```

Comment:
I'm surprised that this is even an option.  I don't remember why this isn't on by default, but maybe it should be (cc @wileyj)